docs/src/notebooks/2_finding_balance.jl

```diff
@@ -168,11 +168,106 @@ vec_soln = parsimonious_flux_balance_analysis_vec(
     model,
     Tulip.Optimizer; # start with Tulip
     modifications = [
-        change_constraint("R_EX_glc__D_e", -12, -12),
+        change_constraint("R_EX_glc__D_e"; lb = -12, ub = -12),
         change_optimizer_attribute("IPM_IterationsLimit", 500), # we may change Tulip-specific attributes here
     ],
     qp_modifications = [
         change_optimizer(OSQP.Optimizer), # now switch to OSQP (Tulip wouldn't be able to finish the computation)
         silence, # and make it quiet.
     ],
 )
+
+# ## Flux balance analysis with molecular crowding (FBAwMC)
```

Comment:
this is from the other PR right?

Reply:
Nope, this is for this PR, I added a modification that enables you to add some functionality to fba.

Reply:
ah okay, better separate it

Reply:
I suppose, but this entire PR is about modifications